server/index.js

```diff
@@ -1,16 +1,20 @@
-const express = require("express");
-const morgan = require("morgan");
-const cors = require("cors");
+const express = require('express');
+const morgan = require('morgan');
+const cors = require('cors');
 const app = express();
+const channel = require('./controller/channel.js');
 // Middleware
 app.use(express.json());
-app.use(morgan("dev"));
+app.use(morgan('dev'));
 app.use(cors());
 
-app.get("/", (req, res) => {
-  res.send("hello world");
+app.get('/', (req, res) => {
+  res.send('example endpoint');
```

Comment:
Please move mock code into a test module.

Reply:
this is OK IMO, it's per the spec in [the ticket for this work](https://trello.com/c/pWfygajy/8-architecture-set-up-api-boilerplate)

Reply:
I see, ignored my last suggestion.